src/langs/json/de.json

```
@@ -347,7 +347,8 @@
     "changeOrg": "Organisation ändern",
     "personalAcct": "Persönliches Konto",
     "signIn": "Anmelden",
-    "uploadEmail": "Hochladen per E-Mail"
+    "uploadEmail": "Hochladen per E-Mail",
+    "donate": "Spenden",
```

This might be the error. JSON objects can't have trailing commas.
```suggestion
    "donate": "Spenden"
```